site/content/docs/contributing/project-structure.md

```diff
@@ -9,17 +9,48 @@ menu:
 
 🚧 This is a work-in-progress 🚧
 
-## CLI
+## Code Layout
+### CLI
+kind's CLI commands are defined in [cmd](/cmd) and are organized as follows:
+
 ```
-.
-├── cmd
-│   └── kind
-│       ├── build/   # Build images
-│       ├── create/  # Create cluster
-│       ├── delete/  # Delete cluster
-│       ├── get/     # List kubeconfigs and clusters
-│       └── kind.go  # Root command
-├── main.go          # Entrypoint
+cmd/
```

Comment:
I think having the files listed will just cause this to go stale very quickly. It's probably more useful to outline where some key logic is like below for EG:
- the main()
- cobra commands
- pkg/cluster
- pkg/cluster/config/...
...

Reply:
so glad you say that, it bothered me a whole lot I did it in the first place.